bazel/cargo/wasmtime/remote/BUILD.rustix-0.35.12.bazel

```diff
@@ -195,8 +191,8 @@ rust_library(
             "@rules_rust//rust/platform:x86_64-linux-android",
             "@rules_rust//rust/platform:x86_64-unknown-freebsd",
         ): [
+            "@wasmtime__libc__0_2_136//:libc",
             "@wasmtime__errno__0_2_8//:errno",
-            "@wasmtime__libc__0_2_126//:libc",
```

Comment:
Nit: Bazel deps should be sorted alphabetically (`errno`, `libc`), otherwise a format checker might complain.